README.md

```diff
@@ -175,6 +185,68 @@ jobs:
           body: ${{ steps.calibre.outputs.markdown }}
 ```
 
+## Combined workflow
+
+You can combine all of the above into one all-encompassing workflow
+
+```yml
+# Compress images:
+# - on pull requests from this report containing images
+# - on pushing of images to master (for forks)
+# - on demand

```

Comment:
What do you mean by on demand?

Reply:
You get an extra button in the Actions tab when you add `workflow_dispatch`. See here: https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/

Reply:
Oh sweet! 

Could you add that link to the line?

`# - on demand (https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/)`

Reply:
Done and also linked in the section above.